python/CCLE_data.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun 20 12:10:43 2019
+
+@author: marcdimeo
+"""
+
+"""
+This document serves as a way to find and manipulate all CCLE data into a dataset
+which can be used for the histone correlation calculations
+"""
+
+import GEOparse
+import mygene
+
+"""
+GETTING GENE NAMES
+"""
+
+file = open("GPL15308.txt", 'r')
+
+gene_ids = []
+
+for line in file:
+    if line.startswith('!') or line.startswith('#') or line.startswith('I') or line.startswith('^') :
+        pass
+    else:
+        ids = line.split("_")[0]
+        gene_ids.append(ids)
+file.close()
```

Comment:
This is a great start, and you are able to get gene symbols with this code. However, if you want to make the code more robust in the future, this needs to be vectorized using numpy or pandas.